AmplifyPlugins/Predictions/AWSPredictionsPlugin/Configuration/ConvertConfiguration.swift

```diff
@@ -0,0 +1,151 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+//import AWSCore
```

Comment:
Leftover commented out import 😅 

It's also in  `IdentifyConfiguration.swift`, `InterpretConfiguration.swift`, and `PredictionsPluginConfiguration.swift`

Reply:
🤦  _grumble linter grumble grumble_
[07e0cf7](https://github.com/aws-amplify/amplify-swift/pull/2895/commits/07e0cf7cec7c31e2c17e9f38e12c26b6ce935dc3)